daemon-tests/tests/main/collaborative_settlement.rs

```diff
@@ -24,12 +24,7 @@ async fn collaboratively_close_an_open_cfd_maker_going_long() {
 async fn maker_rejects_collab_settlement_after_commit_finality() {
     let (mut maker, mut taker) = start_both().await;
     let order_id = open_cfd(&mut taker, &mut maker, OpenCfdArgs::default()).await;
-
-    taker.mocks.mock_latest_quote(Some(dummy_quote())).await;
-    maker.mocks.mock_latest_quote(Some(dummy_quote())).await;
-    let mut quote_receiver = taker.quote_feed().clone();
-    next_with(&mut quote_receiver, |q| q).await.unwrap(); // if quote is available on feed, it propagated through the system
```

Comment:
is this not needed anymore?

Reply:
I've hidden this boilerplate inside `mock_quotes()` method - it's easier to read now.